llama_stack/providers/utils/inference/openai_compat.py

```diff
@@ -1414,12 +1414,12 @@ def b64_encode_openai_embeddings_response(
     for i, embedding_data in enumerate(response_data):
         if encoding_format == "base64":
             byte_array = bytearray()
-            for embedding_value in embedding_data.embedding:
+            for embedding_value in embedding_data["embedding"]:
```

Comment:
careful here. we have lots of paths to these helpers, some pass types others pass dicts. if you can, pass a type.

i agree, the signature hints are probably wrong

Reply:
that's a good idea